README.md

```diff
@@ -1 +1,11 @@
-# 2007-four-horsemen
\ No newline at end of file
+# 2007-four-horsemen
+
+run in terminal: npm i
+
+crate database: lamboshop
+
+open browser to: http://localhost:3000
+or
+open browser to: http://127.0.0.1:3000
+
+run in terminal: node seed/seed.js
```

Comment:
👍

Reply:
I thought it weird to start up an e-commerce app and not have any products immediately available, so I moved products out of the manual seed file. Instead, they are created when the server starts. The 'fake' data is on the seed file though.